Dockerfile

```diff
@@ -9,7 +9,8 @@ MAINTAINER Rithu John <rithu.john@coreos.com>
 # experience when this doesn't work out of the box.
 #
 # OpenSSL is required so wget can query HTTPS endpoints for health checking.
-RUN apk add --update ca-certificates openssl
+RUN apk add --update ca-certificates openssl xmlsec \
```

Comment:
No dependency we have does this and it's generally a red flag when evaluating Go packages. I'll continue looking into go-saml, but generally we require a pure Go implementation.

Since most SAML login flows only use a few XML structs, I'd think we'd prefer just having these structs in dex and doing the XML decoding/signature validation ourselves. dex doesn't need to implement all of SAML